shared/api/event.go

```diff
@@ -64,10 +64,14 @@ func (event *Event) ToLogging() (EventLogRecord, error) {
 			ctx = append(ctx, v)
 		}
 
+		requestor, err := json.Marshal(e.Requestor)
+		if err != nil {
+			return EventLogRecord{}, err
+		}
```

Comment:
that's going to be pretty rough for readability I think... Maybe do `fmt.Sprintf("%s/%s (%s)", requestor.Protocol, requestor.Username, requestor.Address)`?